cloudwash/providers/ec2.py

```diff
@@ -38,6 +62,21 @@ def dry_pips():
             [dry_data['PIPS']['delete'].append(dpip['AllocationId']) for dpip in rpips]
             return dry_data['PIPS']['delete']
 
+        def dry_ocps():
+            all_vpcs = ec2_client.list_networks()
+            for vpc in all_vpcs:
+                tags = vpc.raw.tags
+                if tags:
+                    for tag in tags:
+                        key = tag.get('Key')
+                        if key.startswith('kubernetes.io/cluster/'):
+                            ocp_name = key[22:]
+                            #logger.info("ocp_name: " + ocp_name + " but only " + settings.delete_ocp + "* can be deletes")
+                            discovered = discovery_timestamp(vpc)
+                            if ocp_name.startswith(settings.delete_ocp):
```

Comment:
You need to add `delete_ocp` setting to the settings file. Also understand this means that we will only delete the VMs that startwith `settings.delete_ocp`. So is that the sufficient criteria for deletion ?